src/song_utils.py

```diff
@@ -232,3 +232,41 @@ def get_cover_art(filename: str) -> Optional[File]:
 
     # Create a new discord file from the file pointer and name
     return File(image_bytes_fp, filename=cover_filename)
+
+
+def convert_timestamp_to_seconds(time_str: str):
+    # Converts a time string into seconds. Returns 0 if format is invalid.
```

Comment:
I think this returns `None` if format is invalid, not 0

Reply:
FYI you should mark comment threads as resolved when you fix them so that I can see that they're concluded instead of waiting for my input on something